app/views/shared/menu/_large_menu.html.haml

```diff
@@ -26,6 +26,23 @@
           %a{href: "https://openfoodnetwork.org/au/learn/"}
             %span.nav-primary
               = t 'label_learn'
+      - elsif feature? :about_blog_homepage
```

Comment:
Is this some kind of feature flags implementation? It's the first time I see it

Reply:
Yep, Aus use it. On the server also is added in config/initializers/feature_toggle.rb:
require 'open_food_network/feature_toggle'

module OpenFoodNetwork
  FeatureToggle.class_eval do
    private

    def self.features
      {order_cycles: true,
       about_blog_homepage: true}
    end
  end
end

Reply:
Thanks to Maikel for that tip :-)